bin/resources/GameIndex.yaml

```diff
@@ -38084,7 +38120,7 @@ SLUS-20090:
   region: "NTSC-U"
   compat: 5
 SLUS-20091:
-  name: "4x4 Evolution"
+  name: "4x4 Evo"
```

Comment:
As with the other version, Evo instead of Evolution is kind of subjective, depending on what part of the cover you look at :P

Reply:
Title screen:
https://www.youtube.com/watch?v=ehyTAW6M9gM

Front cover + title screen vs spine: 2-1
"4x4 Evo" is the winner \o/